_data/sites.json

```
@@ -8460,6 +8460,16 @@
         ]
     },
 
+    {
+        "name": "Honeypot.io",
+        "url": "https://app.honeypot.io/profile",
+        "notes": "Go to the profile page, click the cog icon, click 'delete your account' and then 'yes delete'",
+        "difficulty": "easy",
+        "domains": [
+            "honeypot.io"
```

There are more domains:

```suggestion
            "honeypot.io",
            "app.honeypot.io",
            "blog.honeypot.io",
            "cult.honeypot.io"
```

---

Ah my bad!
Thanks for making the change